### ISSUE TITLE
WiFi/Sound dont work if i am close the lid or screen goes black

### ISSUE THREAD

**Issue by ghost** [Grippentech/Asus-E200HA-Linux-Post-Install-Script#5]
If my screen goes black or the i close the lid of my laptop both wifi and audio stops working.

**Comment by Grippy98** [comment]
Yes that seems to be an issue I've run into as well. Building a version of the 4.11 kernel right now. Hopefully this gets fixed by it.

**Comment by damienfern** [comment]
I had the same issue.
The only thing i've found to resolve this is to restart network service manually.
`sudo service network-manager restart`
I'm on Linux Mint 18 (Sound works perfectly thanks to you @Grippentech, just an issue when i plug my headphones but "resolved" by installing pavucontrol and managing the sound settings with it).

**Comment by Grippy98** [comment]
Yeah that makes sense. I need to find a better way to automate the service restart when it crashes.
